test/extended/testdata/deployments/deployment-with-ref-env.yaml

```diff
@@ -27,7 +27,6 @@ spec:
         name: deployment-simple
     spec:
       containers:
-      - image: "openshift/origin-base:latest"
+      - image: openshift/origin-base:v1.3.1
```

Comment:
In what case would the version we have on the machine be older than that on the hub? We build this in every `base_ami` run.